Model/UrlInformation.php

```diff
@@ -248,4 +254,20 @@ public function addAlternateLocale(AlternateLocale $alternateLocale)
 
         return $this;
     }
+
+    /**
+     * @return int
```

Comment:
i think null actually makes sense. otherwise you don't know if depth was
set to 0 or not set.